static/src/javascripts/projects/common/modules/commercial/reader-revenue-banner-selector.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+// (date: 08/04/2020) This should be a temporary solution for changing the order of the reader revenue banners
+
+import { getReaderRevenueRegion } from 'common/modules/commercial/contributions-utilities';
+import { getSync as geolocationGetSync } from 'lib/geolocation';
+import { local } from 'lib/storage';
+
+import { membershipEngagementBanner } from 'common/modules/commercial/membership-engagement-banner';
+import { subscriptionBanner } from 'common/modules/ui/subscription-banners/subscription-banner';
+
+// types
+import type { ReaderRevenueRegion } from 'common/modules/commercial/contributions-utilities';
+import type { Banner } from 'common/modules/ui/bannerPicker';
+
+const currentRegion: ReaderRevenueRegion = getReaderRevenueRegion(
+    geolocationGetSync()
+);
+
+const pageViews: number = local.get('gu.alreadyVisited');
+
+const orderBanners = (region: ReaderRevenueRegion, ...banners): Banner[] => region === 'united-kingdom' ? banners : banners.reverse();
+
+const showFromFourthPageView = (currentPageViews: number, banners: Banner[]): Banner[] => {
```

Comment:
Why is show from fourth page view necessary? Feel like it's adding a layer of complexity to conditionally override `canShow`

Reply:
If contributions configure all their visible banners correctly, it ought not to be necessary anyway.

Reply:
it doesn't look like contributions changes the "minimum number of views before showing" based on region, also the same for subscriptions, so I thought since this is temporary, if we put it all in one place it would make it easier to remove in the future